fe/fe-core/src/main/java/com/starrocks/external/iceberg/io/IcebergCachingFileIO.java

```diff
@@ -0,0 +1,201 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
```

Comment:
If this file is from iceberg repo too, plz add license header.

Reply:
done